src/main/java/network/brightspots/rcv/ClearBallotCvrReader.java

```diff
@@ -48,7 +49,7 @@ void readCastVoteRecords(List<CastVoteRecord> castVoteRecords, String contestId)
     // map for tracking unrecognized candidates during parsing
     Map<String, Integer> unrecognizedCandidateCounts = new HashMap<>();
     try {
-      csvReader = new BufferedReader(new FileReader(this.cvrPath));
+      csvReader = new BufferedReader(new FileReader(this.cvrPath, StandardCharsets.UTF_8));
```

Comment:
just curious: what is the benefit of this?

Reply:
See the commit message where this changed: https://github.com/BrightSpots/rcv/pull/587/commits/b07a23b3193aa1a9ab43b27eb63a7a344464c217.

In particular, this is:
> H I Dm: Found reliance on default encoding in network.brightspots.rcv.ClearBallotCvrReader.readCastVoteRecords(List, String): new java.io.FileReader(String)  At ClearBallotCvrReader.java:[line 51]